src/test/CMakeLists.txt

```diff
@@ -177,6 +177,8 @@ if(ENABLE_IPC)
       ipc_tests.cpp
   )
   target_link_libraries(test_bitcoin bitcoin_ipc_test bitcoin_ipc)
+
+  configure_file(.clang-tidy.in .clang-tidy USE_SOURCE_PERMISSIONS COPYONLY)
```

Comment:
In commit "clang-tidy: Disable `ArrayBound` check in src/ipc and src/test" (5b20d172ca2a46a2b525201b4ff2444f9d415d8c)

Kind of a shame to need this outside of the IPC directory. Maybe there should be a `src/ipc/test` subdirectory like `src/wallet/test` to keep IPC stuff grouped together.

Reply:
> I was thinking of it as a followup for a future PR...

I agree with that.

> ... but I'm actually not sure how big the change would be and whether it might require code changes.

Something like a978c6bdb41fd3f8461908c30bb409a26fa62d47 in https://github.com/hebasto/bitcoin/commits/250925-llvm-tidy/.